app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Middleware;
```

Comment:
Do you want to stick with App namespace?

Reply:
For me it's all the same, I'd stay with the `App` - it's a student project, I'd rather avoid problems with namespace etc.